src/constants/apps.ts

```diff
@@ -10,6 +10,10 @@ export const AppsBindings = {
     IN_POST: '/in_post',
 };
```

Comment:
@mickmister , it is a bit confusing to me that we use AppBindings (plural) for a set of constants and AppBinding(singular) for a data model structure. Typically one uses AppBindings/AppBindingCollection for an ordered/unordered) set of AppBinding objects.

Reply:
@catalintomai Thanks for this. I will go with `AppBindingLocations`, because that's what the constant contains. The plural nature of this name is consistent with other constant groups defined in the mm-redux repo. For example, `PostTypes` is an object that contains several kinds of post types.